pkg/solidity-utils/contracts/helpers/BalancerErrors.sol

```diff
@@ -144,6 +144,7 @@ library Errors {
     uint256 internal constant SWAPS_DISABLED = 327;
     uint256 internal constant CALLER_IS_NOT_LBP_OWNER = 328;
     uint256 internal constant PRICE_RATE_OVERFLOW = 329;
+    uint256 internal constant DISPROPORTIONATE_JOIN_OR_EXIT = 330;
```

Comment:
This is a bit confusing as an error code because Balancer pools typically allow for disproportionate joins/exits. Can we somehow clarify "while paused" without stretching the length of the name beyond what's reasonable?

Reply:
Yes; I'll think of something. INVALID_JOIN_EXIT_WHILE_PAUSED?